_config.yml

```diff
@@ -2,7 +2,7 @@
 title: "Izzie Fitzzie"
 email: "Izzie.Fitzzie@gmail.com"
 description: "Southern Girl's Pursuit of Abs"
-baseurl: "" # the subpath of your site, e.g. /blog/
+baseurl: "http://izziez.github.io/Izzie_Fitzzie" # the subpath of your site, e.g. /blog/
```

Comment:
You should not add domain name as part of baseurl. Baseurl should be `/Izzie_Fitzzie`